src/background-worker/.template.config/template.json

```diff
@@ -33,19 +33,25 @@
     },
     "sources": [
         {
-            "source": "../",
+            "source": "../../",
             "include": [
-                "background-worker/**",
-				"shared/xxENSONOxx.xxSTACKSxx.Shared.Application.CQRS.Abstractions/**",
-				"shared/xxENSONOxx.xxSTACKSxx.Shared.Configuration/**",
-				"shared/xxENSONOxx.xxSTACKSxx.Shared.Configuration.Tests/**",
-				"shared/xxENSONOxx.xxSTACKSxx.Shared.Core/**",
-				"shared/xxENSONOxx.xxSTACKSxx.Shared.Core.Tests/**",
-				"shared/xxENSONOxx.xxSTACKSxx.Shared.Messaging.Azure.ServiceBus/**",
-				"shared/xxENSONOxx.xxSTACKSxx.Shared.Messaging.Azure.ServiceBus.Tests/**",
-				"shared/xxENSONOxx.xxSTACKSxx.Shared.Messaging.Commands/**",
-				"shared/xxENSONOxx.xxSTACKSxx.Shared.Messaging.Events/**",
-				"shared/xxENSONOxx.xxSTACKSxx.Shared.Messaging.Handlers/**"
+                "build/**",
+                "deploy/**",
+                "taskctl.yaml",
+                "stackscli.yml",
+                ".gitignore",
+                ".gitattributes",
+                "src/background-worker/**",
+				"src/shared/xxENSONOxx.xxSTACKSxx.Shared.Application.CQRS.Abstractions/**",
+				"src/shared/xxENSONOxx.xxSTACKSxx.Shared.Configuration/**",
```

Comment:
Is this still meant to be there?

Reply:
Nope! Looks like I missed it in my last PR, need to get rid of it  - i'll create a small PR to do that